src/Client.php

```
@@ -320,7 +328,12 @@ public function query(array $command, ?string $db = null): stdClass|array|int
 
         // CRITICAL: Remove readConcern from any non-first operation in a transaction
         // MongoDB will reject commands with readConcern that have txnNumber but not startTransaction
-        if (isset($command['txnNumber']) && !isset($command['startTransaction']) && isset($command['readConcern'])) {
+        // Or if the command is in the readConcernNotSupporteedCommands array
+        if (((isset($command['txnNumber']) &&  !isset($command['startTransaction']) &&  isset($command['readConcern']))
+        if (
```

PHPStan (2.1.31)

Syntax error, unexpected T_IF on line 333


(phpstan.parse)